modules/i18n/src/translations/en-US/portals/remote-user-stores.ts

```diff
@@ -0,0 +1,245 @@
+/**
+ * Copyright (c) 2024, WSO2 LLC. (https://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+/* eslint-disable max-len */
+
+import { RemoteUserStoresNS } from "../../../models/namespaces/remote-user-stores-ns";
+
+export const remoteUserStores: RemoteUserStoresNS = {
+    form: {
+        fields: {
+            accessType: {
+                label: "Access Type",
+                placeholder: "Select access type",
+                validation: {
+                    required: "Access type is required"
+                }
+            },
+            attributes: {
+                validation: {
+                    required: "Attribute mapping is required"
+                }
+            },
+            connectedUserStoreType: {
+                label: "Remote User Store Type",
+                placeholder: "Select the type of remote user store",
+                validation: {
+                    required: "User store type is required"
+                }
+            },
+            description: {
+                label: "Description",
+                placeholder: "Describe the purpose or scope of this user store"
+            },
+            groupIdMapping: {
+                helperText: "Specify the attribute from the user store that stores the group ID.",
+                label: "Group ID",
+                placeholder: "Ex: groupId",
+                validation: {
+                    required: "Group ID mapping is required when read groups is enabled"
+                }
+            },
+            groupnameMapping: {
+                helperText: "Specify the attribute from the user store that stores the group name.",
+                label: "Group Name",
+                placeholder: "Ex: groupName",
+                validation: {
+                    required: "Group name mapping is required when read groups is enabled"
+                }
+            },
+            name: {
+                label: "User Store Name",
+                placeholder: "ex: MY USER STORE",
```

Comment:
Shall we not maintain placeholder text in upper case? also let's use `e.g.` instead of `ex:`

Reply:
What's the reason behind this?

Reply:
> Shall we not maintain placeholder text in upper case?

Placeholders should represent an common example value for the field and it's not common to use a name with all caps.

> also let's use e.g. instead of ex:

Ex: is used in more casual settings. on a second thought, we might not need eg: or ex: in a placeholder at all.

Reply:
Since the user store name should be in uppercase (handled by the field itself), placeholder value is kept in uppercase.

Reply:
@himeshsiriwardana what's your opinion on having ex: or e.g. in doc team perspective?

Reply:
If we are using the example term, it should be `e.g.` But for names we can remove the `e.g.` term and simply have the name as `MY USER STORE` as it is intuitive. This is what we have followed in other locations as well.

Reply:
Updated 70f200f14255f037a46096205acd8f666919bb76